internal/service/hub/handler/dsl/model/model.go

```diff
@@ -18,6 +18,11 @@ var (
 	// FullNodeCacheKey is the cache key for the full nodes.
 	FullNodeCacheKey = "nodes:full"
 
+	// InvalidRequestCount is the prefix used for cache keys related to storing invalid request counts in the current epoch.
+	InvalidRequestCount = "node:stat:invalid"
+	// ValidRequestCount is the prefix used for cache keys related to storing valid request counts in the current epoch.
+	ValidRequestCount = "node:stat:valid"
```

Comment:
Maybe `node:requests:invalid` or `node:request:count:invalid` would make it easier to add more keys in the future?